core/src/dbs/executor.rs

```diff
@@ -242,6 +245,16 @@ impl<'a> Executor<'a> {
 			let is_stm_kill = matches!(stm, Statement::Kill(_));
 			// Check if this is a RETURN statement
 			let is_stm_output = matches!(stm, Statement::Output(_));
+			// Has this statement returned a value
+			let mut has_returned = false;
+			// Do we skip this statement?
+			if ff_txn
+				&& !matches!(stm, Statement::Commit(_))
+				&& !matches!(stm, Statement::Cancel(_))
+			{
+				debug!("Skipping statement due to fast forwarded transaction");
```

Comment:
Do we need this?

Reply:
I figured to add it in since above we state that the query is executed. Leaving it out could leave people debugging for hours 😅